service/device/registry/test-steps/src/main/java/org/eclipse/kapua/service/device/registry/steps/BrokerSteps.java

```diff
@@ -515,6 +527,34 @@ public void disconnectClient(String clientName) throws Exception {
         }
     }
 
+    @When("I Force Disconnect connection with client id {string}")
+    public void disconnectConnectionWithClientId(String clientId) throws Exception {
+        DeviceConnection deviceConnection = deviceConnectionService.findByClientId(SYS_SCOPE_ID, clientId);
```

Comment:
could I suggest to use a parameter to provide the scopeId? (so the test can be invoked also for children accounts)

Reply:
Since it's generated, would it be possible to specify a scopeId from a cucumber test?  I agree that it would be a good idea to extend this (and other tests) to also check child accounts.  But I think it would require a lot of other changes (i.e. setting up the accounts, connecting with those credentials, etc.) and might be better to do as a separate item.